### PR TITLE
ES2015

### PR DESCRIPTION
**Pull request by remillc**
ES2015 enabled with Babel.

### CODE REVIEW

**Comment by noamamit92** [comment]
@david-gang  - please review and merge

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+	"presets": ["es2015"]
```
Comment (david-gang) [inline .babelrc]: We have already  a definition of the preset in the gulp file itself  at https://github.com/ExLibrisGroup/primo-explore-devenv/blob/master/gulp/tasks/buildCustomJs.js#L27 
Did something specific not work with the current configuration?
Anyway it should not be in both places.

File: package.json [hunk 2]
```diff
@@ -4,41 +4,40 @@
   "description": "",
   "author": "",
   "devDependencies": {
+    "babel-core": "^6.18.2",
```
Comment (david-gang) [inline package.json]: babel-core is already a dependency of gulp-babel. https://github.com/babel/gulp-babel/blob/master/package.json#L35
So unless they move it out to a peer dependency we would like to stick with their version

**Comment by david-gang** [comment]
I got no response till now so i am closing this

**Pull request closed without merging**